python/awwvision/README.md

```diff
@@ -35,6 +38,10 @@ This example uses [Container Engine](https://cloud.google.com/container-engine/)
             --num-nodes 2 \
             --scopes cloud-platform
 
+    You may need to set your zone first, e.g.:
```

Comment:
I think "may" will be confusing for people; how do they know whether they need to set their zone? Can we be more specific about the situations in which people will need to set their zone, or if needed what error message they'll see if they don't?

Reply:
makes sense-- I just went ahead and told them to do it first.